src/webserver/HttpBase.h

```diff
@@ -238,6 +238,27 @@ class HttpBase : public util::Taggable
         if (ec)
             return httpFail(ec, "read");
 
+        auto ip = derived().ip();
+
+        if (!ip)
+        {
+            return;
```

Comment:
you were asking me if we prefer curly brackets or no brackets for one line ifs. in this case i'd drop them as it's just a 'return' statement and not some long line.

Reply:
Same sentiments as Alex here. Just be sure that the one-liner still fits formatting-wise when you run the following command to ensure linting compatibility: `find src unittests -type f \( -name '*.cpp' -o -name '*.h' -o -name '*.ipp' \) -print0 | xargs -0 clang-format-11 -i`